config/database.yml

```diff
@@ -13,5 +13,5 @@ development:
   database: c2_development
 test:
   <<: *default
-  database: c2_test
+  url: postgresql://localhost/c2_test
```

Comment:
Also, maybe worth an inline comment explaining the reason, since the Rails convention is to just have the `database` specified.